instructions/FOSS_staging_list.pl

```diff
@@ -182,7 +182,7 @@
       "dir"         => "zm-help",
       "ant_targets" => undef,
       "stage_cmd"   => sub {
-         SysExec("cp -f -r ../zm-help $CFG{BUILD_DIR}");
+         SysExec("(cd .. && rsync -az --relative --exclude '.git' zm-help $CFG{BUILD_DIR}/)");
```

Comment:
`z` option may be unnecessary when copying between fast local filesystems.

Reply:
found that it's used in other places also so created https://jira.corp.synacor.com/browse/ZCS-12371 ticket to fix it everywhere

Reply:
I got it. Thanks.